### ISSUE TITLE
Can't install NpgsqlFSharpAnalyzer

### ISSUE THREAD

**Issue by MangelMaxime** [Zaid-Ajaj/Npgsql.FSharp.Analyzer#9]
Hello,

I can't install NpgsqlFSharpAnalyzer because paket is complaining about not finding NpgsqlFSharpAnalyzer.Core on NuGet.

Indeed when clicking on NpgsqlFSharpAnalyzer.Core in the dependency list on NuGet I got a 404 not found page.

![image](https://user-images.githubusercontent.com/4760796/91731898-c11a1800-eba7-11ea-9164-28dada9c5130.png)

I added to my `paket.dependencies`

```
group Analyzers
    source https://www.nuget.org/api/v2
    nuget NpgsqlFSharpAnalyzer
```

and then run `dotnet paket update --group Analyzers`.

I can't run `dotnet paket add NpgsqlFSharpAnalyzer --group Analyzers` because it complains about the group `Analysers` not being in the `lock` file. I guess it doesn't want to create it automatically...

I am missing a step or is NpgsqlFSharpAnalyzer.Core really missing from NuGet?

**Comment by Zaid-Ajaj** [comment]
Hi @MangelMaxime, 

Thanks a lot for filing this issue, I think I know why this is happening. Since the 3.11 release, I have added project references to `NpgsqlFSharpAnalyzer` and nuget thinks that those are separate packages but in reality, all those dependant projects like `NpgsqlFSharpAnalyzer.Core` are *embedded* in the nuget package of the analyzer and don't have any nuget references. 

In short, yes. `NpgsqlFSharpAnalyzer.Core` is really missing from nuget and I have to publish a new version of it that doesn't reference external nuget packages since everything is already embedded and ready to go. 

> I cannot publish `NpgsqlFSharpAnalyzer.Core` standalone at the moment because it means I would need to publish 4 different packages (yeah this project is complicated)

**Comment by Zaid-Ajaj** [comment]
Should be fixed as of v3.12.1 :smile: please re-open the issue if the problem persists
